sysutils/xdg-utils/Portfile

```diff
@@ -1,9 +1,13 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           gitlab 1.0
 
+gitlab.instance     https://gitlab.freedesktop.org
+gitlab.setup        xdg xdg-utils v1.2.1
 name                xdg-utils
-version             1.1.3
+version             1.2.1
```

Comment:
When `gitlab.setup` is used correctly, it sets `version` correctly for you so you don't need to do it again.
```suggestion
```
The `name` line should be removed for the same reason.